benchmarks/Tosa/CMakeLists.txt

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+################################################################################
+#                                                                              #
+# Benchmark models for Tosa                                                    #
+#                                                                              #
+# Each module specification should be a list that contains the following       #
+# fields: MODULE_NAME, MODULE_TAGS, MLIR_SOURCE, ENTRY_FUNCTION,               #
+# FUNCTION_INPUTS. See iree_mlir_benchmark_suite definition for details about  #
+# these fields.                                                                #
+#                                                                              #
+################################################################################
+
+#set(MOBILENETSSD_FP32_MODULE
+#  "MobileNetSSD"                  # MODULE_NAME
+#  "fp32"                          # MODULE_TAGS
+#  "https://storage.googleapis.com/iree-model-artifacts/MobileNetSSD.tar.gz" # MLIR_SOURCE
```

Comment:
Could you also include a SHA for the IREE commit generating these models? It helps to

- Track it in the source code here to easily find out which IREE commit it ls from.
- Reproduce the same input MLIR file, as we can find the exact IREE commit for building import binaries.
- Update these input MLIR files, as new ones will have different SHAs. So we can avoid overriding the previous one.

Reply:
Hmm, I'm not entirely sure what you mean by that. I generated them locally (if by "generated" you mean TFLite -> MLIR) -- is there a pipeline/established process I'm missing? Or am I completely misunderstanding the request?